src/dashboard/DashboardView.react.js

```diff
@@ -101,10 +102,8 @@ export default class DashboardView extends React.Component {
         link: '/push/new'
       });
     }
-    // The push UI requires immediate and scheduled push (and some ruby endpoints that we will have to remove)
-    /*
 
-    if (features.push && features.push.storedPushData) {
+    if (features.push && parseServerVersion >= '2.2.8') {
```

Comment:
Got it. I'll change back to `features.push.storedPushData` and submit a PR to Parse Server to update that value.

And, yeah, version string comparison 😞